test-curves/src/bn384_small_two_adicity/g1.rs

```diff
@@ -0,0 +1,47 @@
+use ark_ec::{
+    models::{ModelParameters, SWModelParameters},
+    short_weierstrass_jacobian::*,
+};
+use ark_ff::{field_new, Zero};
+
+use crate::bn384_small_two_adicity::{Fq, Fr};
+
+pub type G1Affine = GroupAffine<Parameters>;
+pub type G1Projective = GroupProjective<Parameters>;
+
+#[derive(Clone, Default, PartialEq, Eq)]
+pub struct Parameters;
+
+impl ModelParameters for Parameters {
+    type BaseField = Fq;
+    type ScalarField = Fr;
+}
+
+impl SWModelParameters for Parameters {
```

Comment:
Do we have any plans for the elliptic curve? Or is it included just in case we want integration tests with mixed radix fields

Reply:
I just dropped it in for convenience.